Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/SonarQubeCategoriseProjectTests.cs

```diff
@@ -0,0 +1,205 @@
+﻿/*
+ * SonarScanner for MSBuild
+ * Copyright (C) 2016-2019 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.IO;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarScanner.MSBuild.Tasks;
+using SonarScanner.MSBuild.Tasks.IntegrationTests;
+using TestUtilities;
+
+namespace SonarScanner.Integration.Tasks.IntegrationTests.TargetsTests
+{
+    [TestClass]
+    public class SonarQubeCategoriseProjectTests
```

Comment:
There are existing tests that call the "categories project" target, but they do it as part of calling multiple other tasks. I decided to try to write a more minimal set of tests that just executed the single target to see if they were simpler to read and write.

I try to move some of the existing tests to this new format in a separate commit.